app/src/main/java/io/foundy/hanstargram/view/posting/PostingActivity.kt

```diff
@@ -35,27 +40,57 @@ class PostingActivity : AppCompatActivity() {
         fun getIntent(context: Context): Intent {
             return Intent(context, PostingActivity::class.java)
         }
+
+        fun getIntent(
+            context: Context,
+            postContent: String,
+            postImage: String,
+            postUuid: String
+        ): Intent {
+            return Intent(context, PostingActivity::class.java)
+                .putExtra("content", postContent)
+                .putExtra("image", postImage)
+                .putExtra("uuid", postUuid)
+        }
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_posting)
 
-        showImagePicker()
-
+        val glide = Glide.with(this)
+        val content = findViewById<EditText>(R.id.image_expression)
```

Comment:
아래의 postContent와 헷갈릴 수 있을거같아 아래와 같이 수정하면 좋겠네요
```suggestion
        val contentEditText = findViewById<EditText>(R.id.image_expression)
```